reference/content-types/index.rst

```diff
@@ -58,15 +58,15 @@ content type:
     color
     date
     email
-    internal_links
+    page_selection
```

Comment:
I think we've ordered the names alphabetically here, so that readers can easier find what they are looking for.